R-packages/evalcast/tests/testthat/test-manage-cards.R

```diff
@@ -0,0 +1,187 @@
+library(tibble)
```

Comment:
`get_predictions()` can produce a list of tibbles. The most obvious way would be by setting `ahead=1:4`. I just want to ensure that this works as well.

Reply:
The tests do have lists of tibbles so I am not sure what you are asking about here.